baselines/common/cmd_util.py

```diff
@@ -102,6 +102,8 @@ def common_arg_parser():
     parser.add_argument('--num_env', help='Number of environment copies being run in parallel. When not specified, set to number of cpus for Atari, and to 1 for Mujoco', default=None, type=int)
     parser.add_argument('--reward_scale', help='Reward scale factor. Default: 1.0', default=1.0, type=float)
     parser.add_argument('--save_path', help='Path to save trained model to', default=None, type=str)
+    parser.add_argument('--video_monitor', help='Save video every x steps (0 = disabled)', default=0, type=int)
```

Comment:
I am not a fan of this argument name... to me, that looks more like a flag that determines if video_monitor is turned on or not. Maybe "save_video_interval"?